test_runner/regress/test_replication_start_subxid_overflow.py

```diff
@@ -0,0 +1,217 @@
+import threading
+import time
+
+from fixtures.neon_fixtures import NeonEnv, wait_for_last_flush_lsn, wait_replica_caughtup
+
+
+def initdb(cur):
+    cur.execute("create table t(pk serial primary key, payload integer)")
+    cur.execute(
+        """create or replace function create_subxacts(n integer) returns void as $$
+              declare
+                 i integer;
+              begin
+                 for i in 1..n loop
+                    begin
+                       insert into t (payload) values (0);
+                    exception
+                       when others then
+                          raise exception 'caught something';
+                    end;
+                 end loop;
+              end; $$ language plpgsql"""
+    )
+
+
+def create_subxacts(env, endpoint, n_subxacts, barrier):
+    con = endpoint.connect()
+    cur = con.cursor()
+    cur.execute("create schema background")
+    cur.execute("set search_path=background")
```

Comment:
I need to create two transactions: with overflowed and non-overflow xids.
Last one can be used to perform various visibility checks. 
But we have to commit first one before replica startup, otherwise in case of overflow replica will have no chance to start.

So I need two tables and so two functions.
And I want to use the same init code for creating both. I thought that using schemas is the most convenient way.